tooling/lib/npm.js

```diff
@@ -9,10 +9,11 @@ const {exec} = require('./async');
 /**
  * Returns the dist-tag for the specified package
  * @param {string} packageName
+ * @param {bool} [includeSamples = false]
  * @private
  * @returns {Promise<string>}
  */
-exports.getDistTag = async function getDistTag(packageName) {
+exports.getDistTag = async function getDistTag(packageName, includeSamples = false) {
```

Comment:
To keep consistent with the other tooling methods, let's put these into a single object instead of 2 separate params. (i.e. `getDistTab({packageName, includeSamples = false}) { }`)

Reply:
Done